src/components/DevicesModaleConfigureView.jsx

```diff
@@ -0,0 +1,338 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import ReactMarkdown from 'react-markdown'
+import cx from 'classnames'
+
+import { FixedDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import Button from 'cozy-ui/transpiled/react/Button'
+import Card from 'cozy-ui/transpiled/react/Card'
+import CardContent from 'cozy-ui/transpiled/react/CardContent'
+import CardHeader from 'cozy-ui/transpiled/react/CardHeader'
+import Checkbox from 'cozy-ui/transpiled/react/Checkbox'
+import FileTypeFolderIcon from 'cozy-ui/transpiled/react/Icons/FileTypeFolder'
+import FormControl from 'cozy-ui/transpiled/react/FormControl'
+import FormControlLabel from 'cozy-ui/transpiled/react/FormControlLabel'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import IconButton from 'cozy-ui/transpiled/react/IconButton'
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import Radio from '@material-ui/core/Radio'
+import RadioGroup from 'cozy-ui/transpiled/react/RadioGroup'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import { isQueryLoading, useClient, useQuery } from 'cozy-client'
+
+import {
+  ROOT_FOLDER_ID,
+  buildFoldersQuery,
+  toCozyDirectory,
+  toCozyOAuthClient,
+  updateDirectoriesExclusions
+} from 'lib/deviceConfigurationHelper'
+
+const hasQueryFailed = query => query && query.fetchStatus === 'failed'
+
+const FoldersLoading = () => (
+  <div className="u-ta-center">
+    <Spinner size="xxlarge" />
+  </div>
+)
+
+const FolderCheckbox = ({ onClick, checked, mixed, disabled, ...props }) => (
+  <IconButton size="small" onClick={onClick}>
+    <Checkbox
+      checked={checked}
+      mixed={mixed}
+      disabled={disabled}
+      disableRipple={true}
+      readOnly
```

Comment:
:question: What is the goal of `readOnly` here?

Reply:
If I remember correctly, I was hoping this would remove the halo around the checkbox. I found it in `cozy-ui` on some checkbox.
It's not documented anywhere so it's probably not doing anything and it apparently does nothing on the halo.